core/src/main/resources/error/error-classes.json

```diff
@@ -1016,7 +1016,7 @@
   },
   "RECURSIVE_PROTOBUF_SCHEMA" : {
     "message" : [
-      "Found recursive reference in Protobuf schema, which can not be processed by Spark: <fieldDescriptor>"
+      "Found recursive reference in Protobuf schema, which can not be processed by Spark by default: <fieldDescriptor>. try setting the option `recursive.fields.max.depth` as 0 or 1 or 2. Going beyond 3 levels of recursion is not allowed."
```

Comment:
Why is 3 or above not alloweded? Seems pretty low. If a customer wants to set the level, they will be conscious. I think it should be at least high single digits to cover most cases. How about 10?

Reply:
@rangadi agree.